Contract Template/admin/initServer.sqf

```diff
@@ -0,0 +1,13 @@
+// Do not Edit this file.
+
+// Log given traits to RPT 70 minutes after mission start.
+[
+    {cba_missiontime >= 4200},
+    {
+        private _playerTraits = [];
+        {
+            _playerTraits pushBack [_x, _y];
+        } forEach tac_supplies_playerTraits;
+        INFO_1("Player Traits: %1",_playerTraits); // For now just logging to RPT, future uses TBD.
+    },
+] call CBA_fnc_waitUntilAndExecute;
```

Comment:
Whyyyy. `waitAndExecute` with 4200 delay.

Reply:
Mission always runs for about an hour before actual start, this is so it logs about 10 minutes after actual start. Is there a better way to do this?

Reply:
Yes, `CBA_fnc_waitAndExecute`.

Reply:
I wonder where my brain went when I wrote this 🤦‍♂️

Reply:
Down some complex pathways!